tests/system/action/committee/test_json_upload.py

```diff
@@ -0,0 +1,298 @@
+from openslides_backend.action.mixins.import_mixins import ImportState
+from openslides_backend.permissions.management_levels import OrganizationManagementLevel
+from tests.system.action.base import BaseActionTestCase
+
+
+class CommitteeJsonUpload(BaseActionTestCase):
+    def test_json_upload_correct(self) -> None:
+        response = self.request(
+            "committee.json_upload",
+            {"data": [{"name": "test", "description": "A long text"}]},
+        )
+        self.assert_status_code(response, 200)
+        assert response.json["results"][0][0]["rows"][0] == {
+            "state": ImportState.NEW,
+            "messages": [],
+            "data": {"name": "test", "description": "A long text"},
+        }
+        self.assert_model_exists(
+            "action_worker/1",
+            {
+                "result": {
+                    "import": "committee",
+                    "rows": [
+                        {
+                            "state": ImportState.NEW,
+                            "messages": [],
+                            "data": {"name": "test", "description": "A long text"},
+                        },
+                    ],
+                },
+            },
+        )
+        assert response.json["results"][0][0]["statistics"][1] == {
+            "name": "Committees created",
+            "value": 1,
+        }
+        assert response.json["results"][0][0]["statistics"][2] == {
+            "name": "Committees updated",
+            "value": 0,
+        }
+
+    def test_json_upload_duplicate_in_db(self) -> None:
+        self.set_models({"committee/7": {"name": "test"}})
+        response = self.request("committee.json_upload", {"data": [{"name": "test"}]})
+        self.assert_status_code(response, 200)
+        assert response.json["results"][0][0]["rows"][0] == {
+            "state": ImportState.DONE,
+            "messages": [],
+            "data": {"name": "test", "id": 7},
+        }
+
+    def test_json_upload_date(self) -> None:
+        response = self.request(
+            "committee.json_upload",
+            {
+                "data": [
+                    {
+                        "name": "test",
+                        "meeting_name": "test meeting",
+                        "start_date": "2023-08-09",
+                        "end_date": "2023-08-10",
+                    }
+                ]
+            },
+        )
+        self.assert_status_code(response, 200)
+        assert response.json["results"][0][0]["rows"][0] == {
+            "state": ImportState.NEW,
+            "messages": [],
+            "data": {
+                "name": "test",
+                "meeting_name": "test meeting",
+                "start_date": 1691539200,
+                "end_date": 1691625600,
+            },
+        }
+
+    def test_json_upload_wrong_date(self) -> None:
```

Comment:
Write one more date test and check, whether only combinations of both date fields filled or not filled are valid, see https://github.com/OpenSlides/openslides-backend/issues/1667.

Reply:
Add start_date xor end_date check and test.